client/scss/components/forms/_input-text.scss

```diff
@@ -38,6 +38,14 @@ textarea {
 .w-field--date_field,
 .w-field--date_time_field,
 .w-field--time_field {
+  // If these fields are in an inline panels' row then allow them to take up their grid size
+  .w-field-row & {
+    input {
+      width: 100%;
+      max-width: 15.625rem;
+    }
+  }
```

Comment:
I’m unclear on why we’d want to do this. Can you say more @stevedya? Is it for the alignment of comment buttons with those fields, or are there other reasons?